src/app/Table.elm

```diff
@@ -11,93 +11,114 @@ import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
 import Types exposing (PlayerID, PlayerName)
-import TableMsg exposing (Msg(..))
 import Dice
+import Identicon exposing (identicon)
 
 
 type alias DiceCount =
     Int
 
 
 type alias TablePlayer =
-    { id : PlayerID
+    { id : Int
     , name : PlayerName
     , diceCount : DiceCount
     }
 
 
+type alias TableConfig msg =
+    { callLiar : Int -> msg
+    }
+
+
 type alias TablePlayers =
     List TablePlayer
 
 
-type alias Model =
+type alias State =
     { tablePlayers : TablePlayers
     , myCup : Dice.Cup
+    , currentPlayerId : Int
     }
 
 
-init : Model
-init =
+emptyState : State
+emptyState =
     { tablePlayers = noPlayers
     , myCup = Dice.emptyCup
+    , currentPlayerId = 0
     }
 
 
-fromPlayersAndCup : List ( Int, String, Int ) -> List Int -> Model
-fromPlayersAndCup tuples cup =
-    let
-        tupleToPlayer ( id, name, diceCount ) =
-            { id = id, name = name, diceCount = diceCount }
-    in
-        { tablePlayers = List.map tupleToPlayer tuples
-        , myCup = Dice.cupFromIntList cup
-        }
-
-
 noPlayers : TablePlayers
 noPlayers =
     []
 
 
-testPlayers : TablePlayers
-testPlayers =
-    [ TablePlayer 1 "Joe" 6, TablePlayer 2 "Garrett" 5 ]
-
-
-view : (Msg -> msg) -> Model -> Html msg
-view privateMsg { tablePlayers, myCup } =
+view : TableConfig msg -> State -> Html msg
+view tableConfig { tablePlayers, myCup, currentPlayerId } =
     div []
         [ h3 [] [ text "Players" ]
-        , playersView tablePlayers
-        , liarButton privateMsg
+        , playersView tableConfig tablePlayers currentPlayerId
         , h3 [] [ text "My Cup" ]
         , Dice.viewCup myCup
         ]
 
 
-liarButton privateMsg =
-    button [ onClick <| privateMsg CallLiar ] [ text "Call Liar!" ]
-
-
-playersView : TablePlayers -> Html msg
-playersView tablePlayers =
+playersView : TableConfig msg -> TablePlayers -> Int -> Html msg
+playersView tableConfig tablePlayers currentPlayerId =
     div [] <|
         List.map
-            playerView
+            (\player -> playerView tableConfig player (currentPlayerId == player.id))
             tablePlayers
 
 
-playerView : TablePlayer -> Html msg
-playerView { id, name, diceCount } =
-    div
-        [ style
-            [ ( "border", "1px solid green" )
-            , ( "width", "200px" )
-            , ( "padding", "10px" )
-            , ( "margin", "8px" )
-            , ( "display", "inline-block" )
+playerView : TableConfig msg -> TablePlayer -> Bool -> Html msg
+playerView { callLiar } { id, name, diceCount } isCurrentPlayer =
```

Comment:
I have a feeling that isCurrentPlayer and lines 79, 85 can be pushed up to the caller so we just have playerView and currentPlayerView which styles players differently with no logic.

Motivation from recent read about removing conditionals or at least pushing it up kinda like side effects.